conf/machine/imx8mpevk.conf

```diff
@@ -0,0 +1,64 @@
+#@TYPE: Machine
+#@NAME: NXP i.MX8MP Evaluation Kit and i.MX8MP Evaluation Kit
+#@SOC: i.MX8MP
+#@DESCRIPTION: Machine configuration for NXP i.MX8MP EVK
+#@MAINTAINER: Alexandru Palalau <ioan-alexandru.palalau@nxp.com>
+
+MACHINEOVERRIDES =. "mx8:mx8m:mx8mp:"
```

Comment:
Please set the BSP to use.

Reply:
Ups, sorry I missed it.

Modifying _local.conf_ is never a good idea... 😄